### DISCUSSION TITLE
Twitter: oauth_token and oauth_token_secret not part of callback

### DISCUSSION THREAD

**Discussion by Hasham-dev** [nextauthjs/next-auth#5851] [Help]
### Provider type

Twitter

### Environment

 System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (4) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
    Memory: 4.94 GB / 15.55 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 16.17.0 - /usr/local/bin/node
    Yarn: 1.22.17 - /usr/bin/yarn
    npm: 8.15.0 - /usr/local/bin/npm
  Browsers:
    Brave Browser: 107.1.45.123
    Chrome: 107.0.5304.110
  npmPackages:
    next: 12.1.0 => 12.1.0 
    next-auth: ^4.15.0 => 4.16.4 
    react: 17.0.2 => 17.0.2 

### Reproduction URL

No Available

### Describe the issue

Not getting the oauth_token & oauth_token_secret after authorization

![image](https://user-images.githubusercontent.com/64362929/202838876-4ccb11d3-ed7e-4a40-a8b9-d9841ebec0df.png)

This is what object I'm getting in return on the JWT callback.

![image](https://user-images.githubusercontent.com/64362929/202838910-ca4cc006-7a10-4013-8e31-4d3aa7ffd9cb.png)

### How to reproduce

I'm using simple snippet for this.

```js
import NextAuth from "next-auth";
import TwitterProvider from "next-auth/providers/twitter";

export const authOptions = {
  // Configure one or more authentication providers
  providers: [
    TwitterProvider({
      clientId: process.env.TWITTER_CLIENT_ID,
      clientSecret: process.env.TWITTER_CLIENT_SECRET,
      version: "2.0", // opt-in to Twitter OAuth 2.0
      authorization: {
        url: "https://twitter.com/i/oauth2/authorize",
        params: {
          scope: "users.read tweet.read tweet.write like.read list.read",
        },
      },
    }),
    // ...add more providers here
  ],
  session: {
    strategy: "jwt",
  },
  callbacks: {
    jwt: ({ token, account, ...props }) => {
      console.log({ token, account, props }, props?.profile?.data);

      if (account?.access_token) {
        token.access_token = account.access_token;
      }
      if (account?.refresh_token) {
        token.refresh_token = account.refresh_token;
      }
      return token;
    },
    session: async ({ session, user, token }) => {
      session.user = user;
      session.token = token;
      return session;
    },
  },
  secret: process.env.NEXTAUTH_SECRET,
};
export default NextAuth(authOptions);

```

### Expected behavior

Expecting oauth_token & oauth_secret_token to be present after authorization request

**Comment by ThangHuuVu** [comment]
Hi there, if you want to get `oauth_token` & `oauth_secret_token`, don't use  `version: "2.0"` in the provider config 👀
